### ISSUE TITLE
Support RTL in Popups

### ISSUE THREAD

**Issue by NicholasBoll** [Workday/canvas-kit#1158]
## 🐛 Bug Report

A Popup inside an RTL `<CanvasProvider>` doesn't render correctly.

## To Reproduce

Go to https://codesandbox.io/s/1gj73

## Expected Behavior

The heading of RTL text should be on the right as well as the body text. Also there should be space between the "OK" and "Cancel" buttons.

![image](https://user-images.githubusercontent.com/338257/125863347-deae6ef5-feda-4caf-a3e7-633b6294d8ea.png)

## Actual Results

Heading and text are on the left and there is no space between the buttons.

![image](https://user-images.githubusercontent.com/338257/125863371-db8f32dd-e5fe-42b5-bac1-7433fdced9d8.png)

## Link to repl or repo (highly encouraged)

Go to https://codesandbox.io/s/1gj73

If you click the `Fix` button, it will add a `dir="rtl"` to the `Modal.Card` which will fix all rendering issues.

## Suggestion

Add `dir` to the `usePopupStack` hook so that direction properly follows portals. React theme context already does which is why the 'X' icon flips correctly, but nothing else does.
